Automattic/wp-calypso
# Revert: Experiment: Add Upgrade button to Masterbar

#### Proposed Changes
Experiment P2: pbxNRc-1qR-p2

This reverts the Masterbar Upsell button experiment added in #61361.

#### Testing Instructions

* Go to a Free site
* Verify that the `Upgrade` button is not visible any longer in the Masterbar.
<img width="420" alt="Screenshot on 2022-07-19 at 10-41-36" src="https://user-images.githubusercontent.com/2749938/179735152-7a110339-d728-4001-a9e9-4e6935eadd56.png">
